templates/block_creator/block_creator.py

```diff
@@ -272,55 +302,61 @@ def report(self):
         - address = string
         """
         self.state.check('status', 'running', 'ok')
-        report = self._get_report()
+        container = self._container_sal
```

Comment:
Is this tested?

the get_report function is not on the container sal at all. Its on the tfchainc sal